config/test.exs

```diff
@@ -28,3 +28,20 @@ config :deep_thought, :deepl, auth_key: "auth_key"
 config :deep_thought, :slack,
   bot_token: "bot_token",
   signing_secret: "signing_secret"
+
+# Secret variables with dummy values
+deepl_auth_key = "auth_key"
+
+config :deep_thought, :deepl, auth_key: deepl_auth_key
+
+slack_bot_token = "bot_token"
+
+# this value is used in test cases
+slack_signing_secret = "secret"
```

Comment:
To keep all tests pass, `"secret"` value needs to be used since this value was used at [here](https://github.com/monstar-lab-oss/deepl-for-slack-elixir/pull/20/files#diff-0635f5bc602b4b0f60ccac9f1fd4940c1a7fa00cc26f179baff261b7b78d5cccL23) and [here](https://github.com/monstar-lab-oss/deepl-for-slack-elixir/pull/20/files#diff-0635f5bc602b4b0f60ccac9f1fd4940c1a7fa00cc26f179baff261b7b78d5cccL33) and so on.
This change reduces readability of testing, so If you have better idea, please fit it 🙏

Reply:
Nothing comes to mind right now, but I think this is still perfectly well readable!